### PR TITLE
feat: update transfers/txHash/:txHash endpoint

### PR DESCRIPTION
**Pull request by tcar121293**
update transfers/txHash/:txHash endpoint
## Description
return array of transfers instead of object

## Related Issue Or Context

Closes: #154

## How Has This Been Tested? Testing details.
e2e tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in chainbridge-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

**Comment by MakMuftic** [comment]
@wainola we need to prepare UI for this change

**Comment by tcar121293** [comment]
> @wainola we need to prepare UI for this change

I think this endpoint is currently not used by the explorer-ui. But let @wainola confirm that

**Comment by wainola** [comment]
> > @wainola we need to prepare UI for this change
> 
> I think this endpoint is currently not used by the explorer-ui. But let @wainola confirm that

we use `/transfer/:transferId` on the detail view. But the url has the txHash. Are we going to change this and use this endpoint @MakMuftic?

**Comment by MakMuftic** [comment]
@wainola I think we should change and use this endpoint instead. I'm not sure if something is missing for this change to be implemented on the UI, @wainola let us know if you think we need to add something else :) 

@tcar121293 then we can merge this as it will not brake anything

File: src/services/transfers.service.ts [hunk 1]
```diff
@@ -75,7 +75,7 @@ class TransfersService {
     return transfer as Transfer
   }
 
-  public async findTransferByTxHash(txHash: string, domainID: number): Promise<Transfer> {
+  public async findTransferByTxHash(txHash: string, domainID: number): Promise<Transfer[]> {
```
Comment (wainola) [inline src/services/transfers.service.ts:78]: would it be more semantic `findTransfersByTxHash`?

**Review by wainola — CHANGES_REQUESTED** [review]

**Review by wainola — APPROVED** [review]

**Comment by wainola** [comment]
> @wainola I think we should change and use this endpoint instead. I'm not sure if something is missing for this change to be implemented on the UI, @wainola let us know if you think we need to add something else :)
> 
> @tcar121293 then we can merge this as it will not brake anything

I can add this to the current explorer UI issue related to the query param

**Pull request merged**